### PR TITLE
Node upgrade

### PR DESCRIPTION
**Pull request by brombomb**
I noticed the build was pushing a year without updates.  This looks perfect for a project I'm working on, so I thought I'd update all the dependencies.

* Upgrade Node to 16
* Upgrade Npm to 8
* Upgrade Puppeteer to 2 => 16
* * Only one major breaking change that I could see
* Update one test for github => google (I think robotos.txt is blocking the rendering of github.com)
* Skip one test due to site, and no comparable replacement working

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+registry=https://artifactory.alteryx.com/artifactory/api/npm/npm/
```
Comment (kimmobrunfeldt) [inline .npmrc:1]: This should be left out of the repo

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+lts/gallium
```
Comment (kimmobrunfeldt) [inline .nvmrc:1]: This too

**Review by kimmobrunfeldt — CHANGES_REQUESTED** [review]
Looks good but adds unnecessary npm config files

**Comment by brombomb** [comment]
Cleaned up

**Pull request merged**